### PR TITLE
Do not filter seeds from datacenters as API does no longer mix them

### PR DESCRIPTION
**Pull request by maciaszczykm**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes https://github.com/kubermatic/dashboard/issues/2468

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: src/app/shared/entity/datacenter.ts [hunk 1]
```diff
@@ -20,7 +20,6 @@ export class CreateDatacenterModel {
 export class Datacenter {
   metadata: Metadata;
   spec: DatacenterSpec;
-  seed?: boolean;
```
Comment (maciaszczykm) [inline src/app/shared/entity/datacenter.ts:23]: @lsviben Can you remove it from the backend too? It is not required anymore.

Comment (lsviben) [reply src/app/shared/entity/datacenter.ts:23]: hmm it should have been removed by this - https://github.com/kubermatic/kubermatic/pull/5562/files

**Comment by maciaszczykm** [comment]
/hold 

It seems to be loading seeds at the moment. Perhaps the backend is not deployed yet.

**Comment by maciaszczykm** [comment]
/assign @kgroschoff 

Can you review in the meantime?

File: src/app/shared/entity/datacenter.ts (hunk 1 above)
Comment (maciaszczykm) [reply src/app/shared/entity/datacenter.ts:23]: Okay then, thanks.

**Comment by maciaszczykm** [comment]
/hold cancel

Dev deployment was updated :)

**Pull request merged**